dev/devicelab/bin/tasks/flavors_test.dart

```diff
@@ -63,6 +63,11 @@ Future<TaskResult> _testInstallDebugPaidFlavor(String projectDir) async {
       'argued --flavor value should not be bundled.');
   }
 
+  if (!assetManifest.containsKey('assets/paid/paid.txt')) {
+    return TaskResult.failure('Assets declared with a flavor not equal to the '
```

Comment:
This is an incredibly cryptic failure message. Can you write this in a way that--should I put out a PR and I see this test fails, I have some clue as to how I broke it?

Also, even if it is correct to use "argue" as the verb form of using a CLI argument, I would say it is not common, and likely many of our users, including those for whom English is not their first language, would understand what "argued --flavor value" means.

Also, isn't this error message inverted?

Reply:
Maybe the error message should be something like, "--flavor paid was passed to flutter install, expected to find assets/paid/paid.txt in the asset bundle at ..."

Reply:
I was wrong when I said "isn't this error message inverted"--but again, I think these messages are too confusing because I *did* re-read the error message several times trying to understand the meaning :)

Reply:
I've made the messages more specific

Reply:
thanks